chatterbot/ext/sqlalchemy_app/models.py

```diff
@@ -65,15 +65,16 @@ class Statement(Base):
     def get_statement(self):
         from chatterbot.conversation import Statement as StatementObject
 
-        statement = StatementObject(self.text, extra_data=self.extra_data)
-        for response in self.in_response_to:
-            statement.add_response(response.get_response())
+        for tag in self.tags:
+            statement = StatementObject(self.text, extra_data=self.extra_data, tags=tag.name)
```

Comment:
I believe there might be a logical error here. Won't this create duplicate statements that each have a single tag? The goal here would be to have a single statement that has multiple tags.